docs/reference/topology-rule.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
How about as an operator manual document?

Reply:
Yes it should be there